packages/app/src/vis-packs/core/line/config.tsx

```diff
@@ -50,13 +50,14 @@ function createStore() {
       }),
       {
         name: 'h5web:line',
-        whitelist: [
-          'curveType',
-          'showGrid',
-          'xScaleType',
-          'yScaleType',
-          'showErrors',
-        ],
+        // @ts-expect-error - `partialize` option has typing issues
+        partialize: (state: LineConfig): Partial<LineConfig> => {
+          return Object.fromEntries(
+            Object.entries(state).filter(
+              ([key]) => !['autoScale'].includes(key)
+            )
+          );
+        },
```

Comment:
This bit is annoying. `whitelist` is deprecated in favour of a generic `partialize`, which can be used to filter out the properties we're not interested in persisting. Unfortunately, the typings aren't quite there yet.